.github/ISSUE_TEMPLATE/epic_release.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Epic Release
+about: Epic Release Checklist
+title: 'Epic Release: [epic name]'
+labels: epic-release-activities
+assignees: ''
+
+---
+
+The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done](https://sap.github.io/spartacus-docs/definition-of-done/)
+
+## General
+
+- [ ] Epic has been tested on a production-like server using Spartacus installation script
+- [ ] Epic has been tested in all supported browsers and devices (Check list on DoD page)
+- [ ] Epic has sufficient end to end (cypress) test coverage
+- [ ] Schematics code deprecations have been provided and tested
+- [ ] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
+- [ ] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
+- [ ] Epic CSS supports directionality (if required)
+- [ ] Epic code scans have been conducted (sonar, whitesource, checkmarx)
+
```

Comment:
Maybe also add general coding best practises, the things @Platonn made the product configurator team aware of: (avoid 'public' explictly, no 'private' methods, avoid un-needed subscriptions on components but use the reactive pattern as far as possible...)

Reply:
Hi @ChristophHi . That is encompassed on the [coding guidelines](https://sap.github.io/spartacus-docs/coding-guidelines/) page, which is referenced on the DoD page.